{
    "os": "Linux-5.4.0-121-generic-x86_64-with-glibc2.31",
    "python": "3.10.4",
    "heartbeatAt": "2022-09-21T07:23:04.964108",
    "startedAt": "2022-09-21T07:23:04.515285",
    "docker": null,
    "gpu": "A100-PCIE-40GB",
    "gpu_count": 1,
    "cpu_count": 128,
    "cuda": null,
    "args": [
        "--experiment",
        "case_augmented_training"
    ],
    "state": "running",
    "program": "/cluster/raid/home/zhivar.sourati/logical-fallacy-identification/AMR_parsers_and_graphs/train.py",
    "codePath": "AMR_parsers_and_graphs/train.py",
    "git": {
        "remote": "git@github.com:usc-isi-i2/logical-fallacy-identification.git",
        "commit": "237b4d675230f4649ad7feefe7cdb914294775ca"
    },
    "email": "zhivarsourati@gmail.com",
    "root": "/cluster/raid/home/zhivar.sourati/logical-fallacy-identification",
    "host": "node01",
    "username": "zhivar.sourati",
    "executable": "/cluster/raid/home/zhivar.sourati/anaconda3/envs/general/bin/python"
}
